specification/src/main/asciidoc/platform/PlatformOverview.adoc

```diff
@@ -1,36 +1,36 @@
 == Platform Overview
 
 This chapter provides an overview of the
-Java™ Platform, Enterprise Edition (Java EE™).
+Jakarta™ EE Platform.
 
 === Architecture
 
 The required relationships of architectural
-elements of the Java EE platform are shown in
-_link:#a45[See Java EE Architecture
+elements of the Jakarta EE platform are shown in
+_link:#a45[See Jakarta EE Architecture
 Diagram]_ . Note that this figure shows the logical relationships of the
 elements; it is not meant to imply a physical partitioning of the
 elements into separate machines, processes, address spaces, or virtual
 machines.
 
 The Containers, denoted by the separate
-rectangles, are Java EE runtime environments that provide required
+rectangles, are Jakarta EE runtime environments that provide required
 services to the application components represented in the upper half of
 the rectangle. The services provided are denoted by the boxes in the
 lower half of the rectangle. For example, the Application Client
-Container provides Java Message Service (JMS) APIs to Application
+Container provides Jakarta Messaging APIs to Application
 Clients, as well as the other services represented. All these services
 are explained below. See
 link:#a84[See Java EE Standard
 Services]”.
 
 The arrows represent required access to other
-parts of the Java EE platform. The Application Client Container provides
-Application Clients with direct access to the Java EE required Database
+parts of the Jakarta EE platform. The Application Client Container provides
+Application Clients with direct access to the Jakarta EE required Database
 through the Java API for connectivity with database systems, the JDBC
-_TM_ API. Similar access to databases is provided to JSP pages, JSF
+_TM_ API. Similar access to databases is provided to Jakarta Server Pages, Jakarta Server Faces
```

Comment:
The "\_TM\_" needs to be replaced with "(TM)" to properly render the Trademark symbol.